ui/src/message_popup/author/question_editing_components/indicators_test.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+import sinon from 'sinon';
+
+import Indicators from './indicators.jsx';
+
+import {RadioButton, RadioButtonGroup} from 'material-ui/RadioButton';
+
+import {testQuestion} from '../../test_fixtures.js';
+import {withIndicator} from '../../transformations.jsx';
+import {indicators} from '../../../data/indicators.js';
+
+function testProps(props){
+  return ({
+    indicator: withIndicator(testQuestion).indicator,
+    onIndicatorChange: sinon.spy()
+  });
+}
+
+describe('<EditingComponent.Inidicators />', ()=>{
+  it('renders all the available indicators', ()=>{
+    const props = testProps();
+    const wrapper = shallow(<Indicators {...props} />);
+    expect(wrapper.find(RadioButtonGroup)).to.have.length(1);
+    expect(wrapper.find(RadioButton)).to.have.length(indicators.length);
+    wrapper.find(RadioButton).forEach((node) => {
+      expect(node.props().value).to.be.oneOf(indicators.map(indicator => indicator.id.toString()));
+    });
```

Comment:
Nice!  One wrinkle with this approach is that the test will still pass if say the first indicator repeated n times.  That's not likely to be a real bug, but you could also map over all the `RadioButton` components and get their `props().value` values, and then compare that to the set of indicatorIds and assert they're equal.  It doesn't matter all that much here, but that's maybe slightly better for other circumstances like this.